config/routes.rb

```diff
@@ -13,6 +12,7 @@
   post '/resources/locator-maps', to: 'offices#index'
   get '/resources/weight-estimator', to: 'weight_estimator#index', as: 'weight_estimator'
   get '/service-specific-information(/:id)', to: 'service_specific_information#show', as: 'service_specific_information'
+  get '/tutorials(/:id)', to: 'tutorials#show', as: 'tutorials'
```

Comment:
In the strictest of Rails-y senses, `as: 'tutorials'` should probably be the singular `as: 'tutorial'` for a `show` action.

Reply:
That makes me feel like I need to create a separate route for /tutorials that goes to tutorials#index which renders show for the first tutorial, and call that one 'tutorials'. Otherwise I need to change the existing tutorials_path calls to tutorial_path which feels wrong without specifying a tutorial...